### ISSUE TITLE
Indentation error for multiline class declaration in ts

### ISSUE THREAD

**Issue by lext-7** [eslint/eslint#11936]
**Tell us about your environment**

* **ESLint Version:** v6.0.1
* **Node Version:** v12.5.0
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**
@typescript-eslint/parser

**Please show your full configuration:**

```js
class ApiChainedMixin<TResponse = any, TContext extends IRequestContext = any>
    implements IApiCrawlerMixin<TResponse, TContext> {
```

**What did you expect to happen?**
The next line of multiline class declaration should have the indent.

**What actually happened? Please include the actual, raw output from ESLint.**
It will raise an error that it should not have an indent.
`Expected indentation of 0 spaces but found 4.eslint(indent)`

**Are you willing to submit a pull request to fix this bug?**
Yes

**Comment by platinumazure** [comment]
Hi @lext-7, thanks for the issue.

Is this reproducible with plain Javascript using the default parser? (The syntax would need to be different, of course.)

The `typescript-eslint` project is working on forking the indent rule, so maybe that will address your issue once it is implemented.
